tests/stdlib/turi.nim

```diff
@@ -174,6 +174,7 @@ template main() =
   block: # removeDotSegments
     doAssert removeDotSegments("/foo/bar/baz") == "/foo/bar/baz"
     doAssert removeDotSegments("") == "" # empty test
+    doAssert removeDotSegments(".") == "." # trailing period
```

Comment:
can you also add a test that exercises the behavior of `removeDotSegments` where it doesn't behave like identity?

Reply:
=> https://github.com/nim-lang/Nim/pull/17064